### DISCUSSION TITLE
Use NetworkManager instead of dhcpd

### DISCUSSION THREAD

**Discussion by tiefenauer** [gitbls/sdm#64] [Q&A]
I want to use NetworkManager instead of dhcpd. This is available (in Bullseye) under `sudo raspi-config > 6 Advanced Options > AA Network Config`.

Is there a way I can instruct SDM to configure the image accordingly? I didn't find a parameter for `--bootset` or in 1piboot.conf.

**Comment by gitbls** [comment]
I've had this on my todo list for a while.  Right now there is the `--netman` switch, but it's woefully incomplete and needs some work. After I get this release out I'll re-learn enough network manager to be dangerous 🙄 and implement some proper Network Manager support. 

If you need something sooner than I finish that, you can always do it in a custom phase script or plugin. I will implement the above as a plugin.

**Comment by gitbls** [comment]
Just a further note on this. The `--netman` switch can be used to enable NM. By "woefully inadequate" I meant that sdm doesn't help in any way, such as adding your pre-built NM configuration files.
